src/Tablebot/Plugins/Cats.hs

```diff
@@ -47,9 +51,21 @@ cat =
 
 -- | @getCatAPI@ is a helper function that turns gets a JSON object that may
 -- contain a cat image. Uses https://docs.thecatapi.com/ for cats.
+-- CATAPI_TOKEN must be provided in .env so that there are no cat limits.
 getCatAPI :: IO (Either String CatAPI)
 getCatAPI = do
-  req <- parseRequest "https://api.thecatapi.com/v1/images/search"
+  initReq <- parseRequest "https://api.thecatapi.com/v1/images/search"
+  loadEnv
+  let api_token = getEnv "CATAPI_TOKEN"
+  dToken <-
+    catchIOError
+      api_token
+      ( \e ->
+          if isDoesNotExistError e
+            then putStrLn "Could not find CATAPI_TOKEN, using no token" >> return ""
+            else ioError e
+      )
```

Comment:
I believe this could be done much more simply with `lookupEnv :: String -> IO (Maybe String)`

Reply:
Could the changes made be reviewed in https://github.com/finnbar/tablebot/pull/15/commits/63e6e4382ed6f83fae3dd7034d946c70f06a9fcf please?